maven-core/src/main/java/org/apache/maven/execution/MavenExecutionRequest.java

```diff
@@ -91,8 +92,17 @@ public interface MavenExecutionRequest {
     // ----------------------------------------------------------------------
 
     // Base directory
+
+    /**
+     * @deprecated use {@link #setTopdir(Path)} instead
```

Comment:
Are the base directory and the top directory different? if so, how? (Please don't answer in github comments. Instead add this info to the code.) If not, why this change?